student-store-ui/src/components/Order/Order.jsx

```
@@ -0,0 +1,33 @@
+import * as React from "react";
+import "./Order.css";
+import { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import axios from "axios";
+import OrderCard from "../OrderCard/OrderCard";
+
+export default function Order() {
+  const [order, setOrder] = useState(null);
+
+  const location = useLocation();
+  const id = location.pathname.split("/")[2];
+
+  useEffect(() => {
+    const fetchOrder = async () => {
+      try {
+        const res = await axios.get(`http://localhost:3001/api/orders/${id}`, {
+          params: { orderId: id },
+        });
+        setOrder(res.data.order);
+      } catch (err) {}
```

This is an anti-pattern; we should at least log the error so it's not swallowed and forgotten/impossible to find and debug.